Packs/Illumio/Integrations/IllumioCore/test_data/test_data_responses.py

```diff
@@ -0,0 +1,72 @@
+from illumio import VirtualService, Reference, ServicePort, TrafficFlow, TrafficNode, Workload, TimestampRange
```

Comment:
This module is not needed. Your lambda functions could just as easily return, e.g. `VirtualService.from_json(success_response)` to avoid duplicating the entire response as a python object.

Reply:
This change is applicable only for VirtualService I believe.

Reply:
Why? Traffic analysis can use a list comprehension to accomplish something similar.

Reply:
so, the `test_data_response.py` can be totally eliminated & we can assert for the JSON response itself.

Reply:
Yes, there shouldn't be a need for duplicate JSON/python objects.

Reply:
@dsommerville-illumio removed the module & updated as per the suggested changes